helm/policies-aws/templates/AWSCAPI.yaml

```diff
@@ -113,3 +113,20 @@ spec:
               {{ `{{` }} regex_replace_all('^$', '{{ `{{` }}@{{ `}}` }}', '{{ .Values.awsClusterRoleIdentity.sourceIdentityRef.name}}') {{ `}}` }}
             kind: |-
               {{ `{{` }} regex_replace_all('^$', '{{ `{{` }}@{{ `}}` }}', '{{ .Values.awsClusterRoleIdentity.sourceIdentityRef.kind }}') {{ `}}` }}
+  # awsmachinetemplate-default-empty-string adds default values to fields in a AWSMachineTemplate CR if it has the watch-filter label and the values are empty strings.
+  # its used to default instanceType for a control plane machine in case its empty
+  - name: awsmachinetemplate-default-empty-string
```

Comment:
Do you only need this for empty strings? Or also if the instanceType is completely missing?

Reply:
what is the differnece? hor how should I do that? i assumed empty string is same as missing